### PR TITLE
BREAKING - Remove LayoutAnimation experimental flag on Android

### PR DESCRIPTION
**Pull request by janicduplessis**
I don't remember exactly where we talked about this but LayoutAnimation on Android is pretty stable now so there's no reason to keep it behind an experimental flag anymore. The only part that is not really stable is delete animations, so what I did is remove the default delete animation that we provide in presets.

**Test plan**
Tested that layout animations work properly without any config.

### CODE REVIEW

File: Libraries/LayoutAnimation/LayoutAnimation.js [hunk 1]
```diff
@@ -110,10 +106,6 @@ var Presets = {
       type: Types.spring,
       springDamping: 0.4,
     },
-    delete: {
-      type: Types.linear,
-      property: Properties.opacity,
-    },
```
Comment (mkonicek) [inline Libraries/LayoutAnimation/LayoutAnimation.js]: Why remove this?

File: Libraries/LayoutAnimation/LayoutAnimation.js [hunk 2]
```diff
@@ -86,10 +86,6 @@ function create(duration: number, type, creationProp): Config {
     update: {
       type,
     },
-    delete: {
-      type,
-      property: creationProp,
-    },
```
Comment (mkonicek) [inline Libraries/LayoutAnimation/LayoutAnimation.js]: Why remove this?

**Comment by mkonicek** [comment]
With the delete animation removed from the config what happens when views are removed? They disappear with no animation?

**Comment by mkonicek** [comment]
Good idea, thanks!

@facebook-github-bot shipit

**Comment by janicduplessis** [comment]
Exactly

**Comment by mkonicek** [comment]
@facebook-github-bot shipit

**Comment by janicduplessis** [comment]
You may have to check for calls to `setLayoutAnimationEnabledExperimental` in FB apps, the docs tells to check if the method exists before calling it but it may not always be the case in practice.

**Comment by mkonicek** [comment]
Good point, thanks @janicduplessis. Will check.

**Comment by janicduplessis** [comment]
@mkonicek Any update on this?

**Comment by mkonicek** [comment]
Sorry! Feel free to ping me on Slack, I don't see GitHub notifications as I have too many :(

**Comment by mkonicek** [comment]
Trying to merge again.

**Comment by mkonicek** [comment]
Hi Janic, people had to revert this because it broke something in the Ads Manager. I asked to explain what it broke exactly. Probably they use `LayoutAnimation` in some code shared between iOS and Android and is started behaving differently on Android :/

https://github.com/facebook/react-native/commit/28ed5eddf2e54146e915ba4427d3960331231bc3

**Pull request closed without merging**